sqldb/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "sqldb"
```

Comment:
By convention, the providers are named for their specific implementation. This crate should be something like `wasmcloud-provider-postgres` - the crate name/artifact name should carry with it the implementation scope.